kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -4789,6 +4789,36 @@ def hotplug_cpu(self, cpu_num):
             raise kvmagent.KvmError(err)
         return
 
+    def _qga_online_hotplugged_cpus(self):
+        """After CPU hotplug, use QGA guest-set-vcpus to online newly added CPUs."""
+        try:
+            qga = VmQga(self.domain)
```

Ruff (0.15.7)

[error] `VmQga` may be undefined, or defined from star imports

(F405)